### ISSUE TITLE
IMA 1.0.1 Stop working on IE 11

### ISSUE THREAD

**Issue by canhnm** [googleads/videojs-ima#470]
Hi,

IMA version 1.0.1 does not work with IE 11 due to error.
Console shows:
```
SCRIPT1010: Expected identifier
File: videojs.ima.js, Line: 2552, Column: 45
```

Which is: 
`Controller.prototype.extend = function(obj, ...args) {`

**Comment by shawnbuso** [comment]
Thanks for the report - we'll take a look at this to get it fixed.

**Comment by ypavlotsky** [comment]
This should be fixed with the release of 1.0.3.
